driver/src/main/com/mongodb/AcknowledgedBulkWriteResult.java

```diff
@@ -120,11 +120,11 @@ public int hashCode() {
     @Override
     public String toString() {
         return "AcknowledgedBulkWriteResult{"
-               + "insertedCount=" + insertedCount
-               + ", matchedCount=" + matchedCount
-               + ", removedCount=" + removedCount
-               + ", modifiedCount=" + modifiedCount
-               + ", upserts=" + upserts
+               + "insertedCount = " + insertedCount
```

Comment:
Please revert this change.  We've been using IntelliJ's default toString generator and this will introduce an inconsistency with all the other generated methods.